sources/editor/Xenko.Assets.Presentation/View/ImageDictionary.xaml

```diff
@@ -1112,6 +1112,20 @@
             </DrawingGroup>
         </DrawingImage.Drawing>
     </DrawingImage>
+  
+    <!-- This folder icon is licensed under creative commons and derived from: Alfredo Hernandez: https://www.alfredocreates.com -->
+    <!-- This one is fairly similar to the one above, the largest difference is that the other one has rounded edges -->
+    <DrawingImage x:Key="FolderIconAlfredo">
+		  <DrawingImage.Drawing>
```

Comment:
Mix of tab and spaces in this line and following (should be space according to rest of file)